### PR TITLE
Add deprecations and FileSeedProvider

### PR DESCRIPTION
**Pull request by jackwaudby**
* S3SeedProvider is deprecated as of 5.26
* FileSeedProvider added in 5.26
* file: is deprecated for URLConnectionProvider as of 5.26
* Reorder providers putting S3SeedProvider last

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

File: modules/ROOT/pages/clustering/databases.adoc [hunk 1]
```diff
@@ -469,50 +469,23 @@ To determine the cause of the problem, it is recommended to look at the `debug.l
...
 ==== Seed providers
 
+The `FileSeedProvider` label:new[introduced in 5.26] supports:
+
+** `file:` 
+
 The `URLConnectionSeedProvider` supports the following:
 
-** `file:`
+** `file:` label:deprecated[deprecated in 5.26]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:478]:
```suggestion
** `file:` label:deprecated[Deprecated in 5.26]
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 2]
```diff
@@ -558,6 +531,40 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
 ======
 =====
 
+The `S3SeedProvider` supports:
+
+** `s3:` label:deprecated[deprecated in 5.26]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:536]:
```suggestion
** `s3:` label:deprecated[Deprecated in 5.26] 
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 3]
```diff
@@ -568,7 +575,7 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
 | URI example
 
 | `file:`
-| `URLConnectionSeedProvider`
+| `URLConnectionSeedProvider` label:deprecated[deprecated in 5.26], `FileSeedProvider` label:new[introduced in 5.26]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:578]:
```suggestion
| `URLConnectionSeedProvider` label:deprecated[Deprecated in 5.26], +
`FileSeedProvider` label:new[Introduced in 5.26] 
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 4]
```diff
@@ -584,15 +591,15 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
 | `\https://myhttp.com/backups/backup1.backup`
 
 | `s3:`
-| `S3SeedProvider`, `CloudSeedProvider`
+| `S3SeedProvider` label:deprecated[deprecated in 5.26], `CloudSeedProvider` label:new[introduced in 5.25]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:594]:
```suggestion
| `S3SeedProvider` label:deprecated[Deprecated in 5.26], +
`CloudSeedProvider` label:new[Introduced in 5.25] 
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 5]
```diff
@@ -584,15 +591,15 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
 | `\https://myhttp.com/backups/backup1.backup`
 
 | `s3:`
-| `S3SeedProvider`, `CloudSeedProvider`
+| `S3SeedProvider` label:deprecated[deprecated in 5.26], `CloudSeedProvider` label:new[introduced in 5.25] 
 | `s3://mybucket/backups/backup1.backup`
 
 | `gs:`
-| `CloudSeedProvider`
+| `CloudSeedProvider` label:new[introduced in 5.25]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:598]:
```suggestion
| `CloudSeedProvider` label:new[Introduced in 5.25] 
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 6]
```diff
@@ -584,15 +591,15 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
...
 | `s3://mybucket/backups/backup1.backup`
 
 | `gs:`
-| `CloudSeedProvider`
+| `CloudSeedProvider` label:new[introduced in 5.25] 
 | `gs://mybucket/backups/backup1.backup`
 
 | `azb:`
-| `CloudSeedProvider`
+| `CloudSeedProvider` label:new[introduced in 5.25]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:602]:
```suggestion
| `CloudSeedProvider` label:new[Introduced in 5.25] 
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 7]
```diff
@@ -469,50 +469,23 @@ To determine the cause of the problem, it is recommended to look at the `debug.l
...
-The `S3SeedProvider` supports:
 
-** `s3:`
 
-[NOTE]
-====
-Neo4j 5 comes bundled with necessary libraries for AWS S3 connectivity.
-Therefore, if you use `S3SeedProvider`,`aws cli` is not required but can be used with the `CloudSeedProvider`.
-====
+The `CloudSeedProvider` label:new[introduced in 5.25] supports:
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:485]:
```suggestion
label:new[Introduced in 5.25], the `CloudSeedProvider` supports:
```

File: modules/ROOT/pages/clustering/databases.adoc [hunk 8]
```diff
@@ -469,50 +469,23 @@ To determine the cause of the problem, it is recommended to look at the `debug.l
 
 ==== Seed providers
 
+The `FileSeedProvider` label:new[introduced in 5.26] supports:
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/databases.adoc:472]:
```suggestion
label:new[Introduced in 5.26], the `FileSeedProvider`  supports:
```

**Review by NataliaIvakina — APPROVED** [review]
@jackwaudby, thank you! This looks good. I added a few suggestions to be in line with the rest of the manual.
Please, do not merge this PR! We have to publish the 5.25 docs first.

**Comment by jackwaudby** [comment]
@NataliaIvakina thank you, I've committed the suggestions. Ok sure thing, I will leave it up to you to hit merge when the time comes

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1909.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1909.surge.sh/operations-manual/5/clustering/databases)

**Pull request merged**